src/main/java/chess/tools/Chess.java

```diff
@@ -0,0 +1,149 @@
+package chess.tools;
+
+import chess.tools.game.*;
+import chess.tools.model.ChessModel;
+import chess.tools.move.MoveManager;
+import chess.tools.move.MoveTuple;
+import chess.tools.move.Position;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+public class Chess {
+
+    private ChessModel chessModel;
+    private MoveManager moveManager;
+    private Logger logger = Logger.getLogger(Chess.class.getName());
+    private CommandParser parser = new CommandParser();
```

Comment:
- [ ] rename parser to commandParser, therefore we see the type of identifier in it's name